automations/maintain-changelog/workflow.yaml

```diff
@@ -0,0 +1,20 @@
+name: "📝 Maintain Changelog"
+
+on:
+  release:
+    types: [created]
+    
+jobs:
+  update-changelog:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Update CHANGELOG.md
+        uses: PR-Pilot-AI/smart-actions/quick-task@v1
+        with:
+          api-key: ${{ secrets.PR_PILOT_API_KEY }}
+          agent-instructions: |
+            1. Check out the repository.
+            2. Identify the previous release tag and the current release tag from the trigger event.
+            3. Assemble all commit messages between the new release and the previous one.
```

Comment:
/pilot This should be done in previous steps by the workflow itself using git on the  command line . The result should be passed into `agent-instructions` as a variable